src/AggregateFunctions/QuantileTDigest.h

```diff
@@ -195,15 +195,6 @@ class QuantileTDigest
             BetterFloat l_count = l->count;
             while (r != centroids.end())
             {
-                if (l->mean == r->mean) // Perfect aggregation (fast). We compare l->mean, not l_mean, to avoid identical elements after compress
```

Comment:
Need to clarify what exactly has been changed.
It will be better if you will provide illustructive "textbook" examples on this matter directly in comments.

Reply:
Done.